modules/modules.test.js

```diff
@@ -1,4 +1,4 @@
-import { convertToMorse } from "./modules.js";
+import { convertToMorse, convertToEnglish } from "./morseUtils.js";
```

Comment:
This file will need to be renamed. 
Test files need to be named after the files they are testing (ie if you are testing functions contained in morseUtils.js then this needs to be named morseUtils.test.js)

Reply:
Changed modules.test.js to morseUtils.js to follow best practice